src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java

```diff
@@ -794,6 +797,31 @@ public void testJavaParseExceptionSeverityDefaultError() throws Exception {
         verify(checker, files, expectedViolation);
     }
 
+    @Test
+    public void testTreeWalkerCheckSortingById() {
+        final AbstractCheck check1 = new ConstantNameCheck();
+        check1.setId("zebra");
+
+        final AbstractCheck check2 = new ConstantNameCheck();
+        check2.setId("apple");
+
+        final AbstractCheck check3 = new ConstantNameCheck();
+        check3.setId("mango");
```

Comment:
Please add modules to header comment of InputTreeWalkerSorting
.
Please review how our general test are working, the same verifyxxxxx method should be used